dev/tools/gen_defaults/lib/color_scheme_template.dart

```diff
@@ -3,86 +3,97 @@
 // found in the LICENSE file.
 
 import 'template.dart';
+import 'token_logger.dart';
 
 class ColorSchemeTemplate extends TokenTemplate {
-  ColorSchemeTemplate(super.blockName, super.fileName, super.tokens);
+  ColorSchemeTemplate(this._colorTokensLight, this._colorTokensDark, super.blockName, super.fileName, super.tokens);
```

Comment:
Just for my education, why do we use private properties in constructor?

Reply:
It's no different than the following

ColorSchemeTemplate(
    Map<String, dynamic> colorTokensLight,
    ...
) : _colorTokensLight = colorTokensLight;